stm/stm.ml

```diff
@@ -65,7 +65,7 @@ end
 
 (* During interactive use we cache more states so that Undoing is fast *)
 let interactive () =
-  if !Flags.ide_slave || !Flags.print_emacs || not !Flags.batch_mode then `Yes
+  if !Flags.ide_slave || not !Flags.batch_mode then `Yes
```

Comment:
Is `-emacs -batch` supported?  What would it mean?  If it's not, there should be an error message, right?  If it is, then this changes semantics, right?